src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -619,6 +619,17 @@ default boolean isBeaconPayment(ItemStack stack)
         return Tags.Items.BEACON_PAYMENT.contains(stack.getItem());
     }
 
+    /**
+     * Whether this Item can be used as a payment to activate the vanilla beacon.
```

Comment:
This looks like a copy-paste error:
```suggestion
     * Whether this Item can be used to hide player head for enderman.
```

Reply:
It's change in the last commit